libglusterfs/src/run_fork.c

```diff
@@ -0,0 +1,570 @@
+/*
+  Copyright (c) 2008-2012 Red Hat, Inc. <http://www.redhat.com>
+  This file is part of GlusterFS.
+
+  This file is licensed to you under your choice of the GNU Lesser
+  General Public License, version 3 or any later version (LGPLv3 or
+  later), or the GNU General Public License, version 2 (GPLv2), in all
+  cases as published by the Free Software Foundation.
+*/
+
```

Comment:
```c
/*
 * Legacy implementation of the runner_* API for plaftorms with broken posix_spawn()
 */
 ```
 
 Please add this comment (feel free to extend or improve wording).

Reply:
Ack.